website/aws_helpers.py

```diff
@@ -23,15 +23,15 @@ def s3_querylog_transmitter_from_env():
     return make_s3_transmitter(config.config["s3-query-logs"])
 
 
-def s3_parselog_transmitter_from_env():
+def s3_parselog_transmitter_from_env(config_key="s3-parse-logs"):
```

Comment:
This was called "parselog_transmitter" because it was only used for logs that have to do with parsing. If we're going to make it more generic and have it be about logs that don't have to do with parsing, you can rename it more generically to `s3_log_transmitter_from_env` or something.